index.bs

```diff
@@ -315,62 +322,69 @@ spec: webidl; urlPrefix: https://heycam.github.io/webidl/
       objects indicating the expected state of the <a>budget</a> at given times in the future.
     </p>
     <p>
-      When invoked, it MUST return <a>a new promise</a> <var>promise</var> and run the following
-      steps <a>in parallel</a>:
+      When invoked, it MUST run the following steps:
     </p>
     <ol>
       <li>
+        Let <var>promise</var> be <a>a new promise</a>.
+      </li>
+      <li>
         Let <var>origin</var> be the <a>entry settings object</a>'s <a>origin</a>.
       </li>
       <li>
         If the <var>origin</var> is not a <a>secure context</a>, reject <var>promise</var> with a
         <code>{{SecurityError}}</code> and terminate these steps.
       </li>
       <li>
-        Let <var>details</var> be a new <code>{{sequence}}</code>.
-      </li>
-      <li>
-        If there are entries in <var>origin</var>'s <a>list of budget expectations</a>, for each
-        <var>entry</var>:
-
+        Return <var>promise</var> and run the following step <a>in parallel</a>:
         <ol>
           <li>
-            Let <var>state</var> be a new <code>{{BudgetState}}</code> instance.
+            Let <var>details</var> be a new <code>{{sequence}}</code>.
```

Comment:
Also unrelated, but using https://infra.spec.whatwg.org/#list would be more idiomatic I think, and Web IDL says "Any list can be implicitly treated as a sequence<T>, as long as it contains only items that are of type T." while linking to infra.